doc/index.rst

```diff
@@ -62,3 +62,12 @@ Amazon provides implementations of some common machine learning algortithms opti
     factorization_machines
     lda
     ntm
+
+SageMaker Workflow
+------------------
+SageMaker APIs to export configurations to create and manage Airflow workflows.
```

Comment:
s/to create and manage/for creating and managing

Reply:
Updated!